### ISSUE TITLE
seed-extension heuristic

### ISSUE THREAD

**Issue by martin-raden** [BackofenLab/IntaRNA#39]
### idea (Martin+Fabrizio)
- given that there are only *few* seed interactions possible (to be evaluated)
- for each seed:
  - extend max to the left (right boundary fixed = seed end)
  - extend the best left extension to the right (left boundary fixed = left extension end)
  - extend intervals of best such interaction and do optimal O(n^4) prediction for these intervals only given the single seed
- evaluate performance/ prediction quality
### to check
- master thesis by [M. El-Shaer](http://www.bioinf.uni-freiburg.de/Lehre/Theses/BA_Mohamed_Ezz.pdf) = extension left and right (but not in parallel)

### already done in
- RIblast (Fukunaga, 2017)

**Comment by martin-raden** [comment]
assigned to @Fulanko

**Comment by martin-raden** [comment]
@Fulanko: please also check the BA thesis referenced above.. Thanks!

**Comment by martin-raden** [comment]
open problem: suboptimal enumeration with seed-extension predictors
